minifi-assembly/pom.xml

```diff
@@ -113,12 +113,11 @@ limitations under the License.
         </dependency>
         <dependency>
             <groupId>org.apache.nifi.minifi</groupId>
-            <artifactId>minifi-framework-nar</artifactId>
-            <type>nar</type>
+            <artifactId>minifi-framework-core</artifactId>
         </dependency>
         <dependency>
             <groupId>org.apache.nifi.minifi</groupId>
-            <artifactId>minifi-provenance-reporting-nar</artifactId>
```

Comment:
Since we're switching to the NiFi provenance reporting task, the class name on line 474 of ConfigTransformer needs to be changed as well.